controllers/admin/AdminCmsCategoriesController.php

```diff
@@ -221,11 +221,11 @@ public function renderForm()
                     'name' => 'name',
                     'required' => true,
                     'lang' => true,
-                    'hint' => $this->l('Invalid characters:').' &lt;&gt;;=#{}'
+                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Notifications.Error').' &lt;&gt;;=#{}'
```

Comment:
I see it as an hint and not a DisplayError or something error-related, to me it's an indication to let you know which characters you should avoid (we agreed the wording is not the best).